.github/workflows/copy-linked-issue-labels.yml

```diff
@@ -0,0 +1,21 @@
+name: Copy labels from linked issues
+on:
+  pull_request:
+    types: [opened, edited, review_requested, synchronize, reopened, ready_for_review]
```

Comment:
Should be `pull_request_target` since you'll need write permissions to update the PR. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

- Recent change that needed similar permissions  https://github.com/opensearch-project/OpenSearch/pull/9374